### PR TITLE
Remove engine.Job from diff

### PR DESCRIPTION
**Pull request by runcom**
Part of: #12151

I'll happly work on another of this if there's need and this is ok ^.^

Signed-off-by: Antonio Murdaca me@runcom.ninja

### CODE REVIEW

File: api/server/server.go [hunk 1]
```diff
@@ -351,10 +351,24 @@ func getContainersChanges(eng *engine.Engine, version version.Version, w http.Re
 	if vars == nil {
 		return fmt.Errorf("Missing parameter")
 	}
-	var job = eng.Job("container_changes", vars["name"])
-	streamJSON(job, w, false)
 
-	return job.Run()
+	d := getDaemon(eng)
+	cont, err := d.Get(vars["name"])
```
Comment (duglin) [inline api/server/server.go]: I think we may want to check to see if "name" is empty to return a nicer error message.

File: api/server/server.go [hunk 2]
```diff
@@ -351,10 +351,24 @@ func getContainersChanges(eng *engine.Engine, version version.Version, w http.Re
...
+	if err != nil {
+		return err
+	}
+
+	changes, err := cont.Changes()
+	if err != nil {
+		return err
+	}
+
+	w.Header().Set("Content-Type", "application/json")
```
Comment (duglin) [inline api/server/server.go]: Let's try to use WriteJSON() instead so if we need to make a global change to how to write stuff its all in one spot.

File: api/server/server.go (hunk 1 above)
Comment (runcom) [inline api/server/server.go]: :+1: thanks there wasn't any check I'll add one now!

File: api/server/server.go (hunk 2 above)
Comment (runcom) [inline api/server/server.go]: Thanks! haven't looked at that before!

**Comment by runcom** [comment]
@duglin made your proposed changes

**Comment by duglin** [comment]
thanks LGTM

**Comment by icecrime** [comment]
Thanks! LGTM

**Pull request merged**